src/components/loginSignup/loginsignup.css

```diff
@@ -0,0 +1,202 @@
+.login {
+  /* width: 20%;
+  position: absolute;
+  top: 30%;
+  left: 43%;
+  /* border: 1px solid; */
+}
+
+.logoDetail {
+  position: absolute;
+  width: 45%;
+  height: 100vh;
+  left: 0px;
+  top: 0px;
+  background: linear-gradient(180deg, #f0a500 0%, #cf7500 100%);
+  opacity: 0.8;
+}
+
+.backgroundImg {
+  width: 100%;
+  height: 100%;
```

Comment:
Can you set background image here? 
Then it won't be draggable

Reply:
dindt get u

Reply:
If you drag the image now it will come as a draggable in browser.
If you set via `background-image`, then it won't show the behaviour.